docs/src/documentation/guides/technical-setup/README.md

```diff
@@ -39,27 +50,46 @@ Ein Beispiel für eine Konfigurationsdatei befindet sich unter [Konfiguration lo
 Diese konfiguriert den API Gateway Port, die Route zum Backend Service auf dem lokalen Rechner und die URLs des
 für die Entwicklung lokal gestareten Frontends.
 
+### Backend
+Für die lokale Umgebung erstellt man die Konfigurationsdatei
+
+    src/main/resources/application-local.yml
+
+Der Backend Service startet standardmäßig mit dem Port `8080` und sollte in der lokalen Umgebung umgestellt werden.
+
+Zusätzlich wird die Datenbank, der S3 Storage mit zulässigen MIME-Types für die Dokumente und
```

Comment:
Die Beschreibung bezüglich des S3-Storage sollte angepasst werden.

Die Konfiguration wird für die Anbindung an das Deployment der [digiwf-s3-integration](https://github.com/it-at-m/digiwf-core/tree/dev/digiwf-integrations/digiwf-s3-integration) erstellt. 

Bei den Mime-Types handelt es sich um die Mime-Types die beim Dateihandling erlaubt sind.